frontend/src/app/modules/job-status/job-status-modal/job-status.modal.ts

```diff
@@ -13,6 +13,7 @@ import {
 import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 import {JobStatusEnum, JobStatusInterface} from "core-app/modules/job-status/job-status.interface";
 import {NotificationsService} from "core-app/modules/common/notifications/notifications.service";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/cfe89dd706c72154281f341454501767d598e770/frontend/src/app/modules/job-status/job-status-modal/job-status.modal.ts#L18"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>